Utilities/CodeGeneration/EdFi.Ods.CodeGen/Mustache/Resources.mustache

```diff
@@ -481,22 +481,36 @@ namespace {{ResourceClassesNamespace}}
         {{#Referenced}}
         {{#Property}}
 
+        {{#AllIncomingAssociationsAreOptional}}
+        private {{PropertyType}} _{{PropertyFieldName}};
```

Comment:
We might be better off using `JsonPropertyName` instead of `PropertyFieldName` to render the variable's name.

Reply:
I think this is done now, please confirm.